packages/foam-vscode/syntaxes/injection.json

```diff
@@ -5,7 +5,13 @@
     {
       "contentName": "string.other.link.title.markdown.foam",
       "begin": "\\[\\[",
-      "end": "\\]\\]"
+      "beginCaptures": {
+        "0": { "name": "punctuation.definition.metadata.markdown" }
+      },
+      "end": "\\]\\]",
+      "endCaptures": {
+        "0": { "name": "punctuation.definition.metadata.markdown" }
+      }
```

Comment:
This is what the delimiters in `[..](...)` are.

Reply:
basically you are making sure that `[[` and `[` use the same style, am I reading this right?

Reply:
`punctuation.definition.metadata.markdown` is the scope for `[`, `]`, `(` and `)` in `[..](...)`. And I'm adding the `.foam` suffix so that it can be configured for foam in a different way if so desired.